web/lib/utils/axios/checkStatus.ts

```diff
@@ -39,12 +38,13 @@ export function checkStatus(status: number, msg: string, errorMessageMode: Error
       break
 
     case 401:
+      // ** reserve error message
       errMessage = msg || 'The user does not have permission (token, user name, password error or expired)!'
 
       localStorage.removeItem('accessToken')
-      localStorage.removeItem('version')
+      localStorage.removeItem('authParams')
 
-      Router().push('/ui/login')
+      window.location.href = '/ui/login'
```

Comment:
<img width="729" alt="image" src="https://github.com/datastrato/gravitino/assets/9210625/a39258ce-eebf-4cea-a4c8-5910dc00e05e">
For Server Components, use the redirect function instead.